pytest_fixtures/rh_cloud.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from nailgun import entities
+
+from robottelo import manifests
+from robottelo.api.utils import upload_manifest
+from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
+
+
+@pytest.fixture(scope='module')
+def organization_ak_setup(module_org):
+    """A module-level fixture to create Organization and Activation key for tests"""
```

Comment:
This fixture does not create an org, it calls `module_org` and I wonder could you not use the existing `module_manifest_org` ?

Reply:
@swadeley updated the pr.

Reply:
great